client/ayon_core/hosts/max/plugins/publish/validate_mesh_has_uv.py

```diff
@@ -0,0 +1,44 @@
+
+import pyblish.api
+from ayon_core.hosts.max.api.action import SelectInvalidAction
+from ayon_core.pipeline.publish import (
+    ValidateMeshOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from pymxs import runtime as rt
+
+
+class ValidateMeshHasUVs(pyblish.api.InstancePlugin,
+                         OptionalPyblishPluginMixin):
+
+    """Validate the current mesh has UVs.
+
+    It validates whether the current mesh has texture vertex(s).
+    If the mesh does not have texture vertex(s), it does not
```

Comment:
The added (s) here after the word is weird because the plural of vertex is vertices. I'd just write it as:
```suggestion
    It validates whether the current mesh has texture vertices.
    If the mesh does not have texture vertices, it does not
```